__tests__/fixtures/rsc/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "dependencies": {
+    "next": "*",
```

Comment:
pinning next to `13` might be a good idea, given the [server components](https://nextjs.org/docs/advanced-features/react-18/server-components)
(or at least a comment somewhere stating the expectation that Next >=13 gets used)

Reply:
Updated 👍